js/app.js

```diff
@@ -40,87 +28,233 @@ var dogDuck = 0;
 var dragon = 0;
 var pen = 0;
 var petSweep = 0;
-var scissor = 0;
+var scissors = 0;
 var shark = 0;
 var tauntaun =0;
 var unicorn = 0;
 var usb = 0;
 var waterCan = 0;
 var wineGlass = 0;
 
-productImagesParent.addEventListener('render', function(render) {
+var productImageParent = document.getElementById('productImage');
+var responseParent = document.getElementById('response');
+
+function setup() {
+  //think about what does setup do
+  //name var = element(object)
+  productImage1 = generateRandomProduct();
+  productImage2 = generateRandomProduct();
+  productImage3 = generateRandomProduct();
+//TODO make random not equal last 3 nor other 2
+  if (attempts) {
+    productImageParent.removechild(productImageParent.lastChild);
+    productImageParent.removechild(productImageParent.lastChild);
+    productImageParent.removechild(productImageParent.lastChild);
+  }
+  // for (var p = 0; p < 3; i++) {
+  //   productNames ;
+  // }
+  renderProductImage(productImage1);
+  renderProductImage(productImage2);
+  renderProductImage(productImage3);
+  // updateAnswer();
+  // updateAttempts();
+}
+setup();
+renderResponse();
+data();
+buttonLogic();
+
+
+productImageParent.addEventListener('render', function(render) {
   var display = event.target.getAttribute('id');
+  render(display);
   //same logic as below? Find a way to loop this to make code dry
-})
-productImagesParent.addEventListener('click', function(event){
+});
+
+
+productImageParent.addEventListener('click', function(event) {
+  if (attempts === maxAttempts) {
+    return;
+  }
+
   var answer = event.target.getAttribute('id');
-  console.log('id');
-  if (answer != []) {
-    alert('please click on image')
-  } if  (answer === [0]) {
-    console.log(babySweep +=  1);
+  attempts++;
+//for i statement - constructor
+  if (answer === [0]) {
```

Comment:
Could you cut this down by using a for loop? Or a .includes()? That would make your code much DRYer. Also, what are you referencing? What array at position 0-19? You have an error in your console so this is never being seen. But you can't just reference [0]. You need to tell the JS the array you are targeting at that position.